TaylorFramework/Modules/scissors/ExtendedComponentExtensions.swift

```diff
@@ -0,0 +1,167 @@
+//
+//  ExtendedComponentExtensions.swift
+//  Taylor
+//
+//  Created by Alexandru Culeva on 4/12/16.
+//  Copyright © 2016 YOPESO. All rights reserved.
+//
+
+import Foundation
+
+extension ExtendedComponent {
+    
+    func perform(@noescape block: ExtendedComponent -> ()) {
```

Comment:
Could you please come up with a name that suggests function behaves recursively?